scripts/run-local-express.sh

```diff
@@ -1,8 +1,14 @@
 #!/bin/bash
 _dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+export MIGMETA_FILE="$_dir/../tmp/migmeta.json"
+# export VIEWS_DIR="$_dir/../public"
```

Comment:
Pretty sure we don't need `VIEWS_DIR` since the `index.html.ejs` just gets dumped into `dist` with the rest of the bundle. Maybe that's a mistake, not sure. If you want we could add this back and update webpack to put it there, and express to find it there.